util/scripts/launch.sh

```diff
@@ -72,8 +73,5 @@ fi
 gradlew="${base}/gradlew"
 
 # Launch the tool.
-if [ $# -eq 0 ]; then
-    "${gradlew}" -p "${base}" "cli:${tool}:run"
-else
-    "${gradlew}" -p "${base}" "cli:${tool}:run" --args="$*"
-fi
+"${gradlew}" --quiet assemble ":cli:${tool}:assemble"
```

Comment:
Using `--quiet` is a very good idea! However, by dropping the `-p` it is now not possible anymore to run `lfc-dev` from an arbitrary directory. For me, this was the main reason for using the script instead of the corresponding gradle run command. I will provide a fix in another PR.